spec/crc_examples.rb

```diff
@@ -25,6 +25,13 @@
     expect(crc.checksum).to be == expected.to_i(16)
   end
 
+  it "should be the length of the result of pack, based on width" do
+    mustsize = (described_class::WIDTH / 8.0).ceil
+
+    expect(described_class.pack(0).bytesize).to be == mustsize
+    expect(described_class.pack(-1).bytesize).to be == mustsize
```

Comment:
The specs should test the contents of the Strings and the order of the bytes. This could probably be done using a `case`/`when` statement of `described_class::WIDTH` and testing whether `.pack` returned the same value as `[value].pack('L<')`, `[value].pack('S<')`, etc.

Reply:
Thank you, I understand.
However, the previous `.pack` is expressed in big endian and handles 3 bytes, such as CRC24, etc., so I guess I need to take that into consideration.
Anyway, I will fix it.

Reply:
@dearblue I think `Digest::CRC1::WIDTH` should be `1`. You could override `.pack` specifically for `Digest::CRC1` since it's an edge-case unlike `CRC32` or `CRC24`.

Reply:
I have updated it with the comment of <https://github.com/postmodern/digest-crc/commit/e9a249bd04768b97c0f2d9430d3f1a892843e94b#r120884002>.